tests/ci/azure_pipeline_test/dsvm_nightly_linux_cpu_master.yml

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think we are adding files that are not needed, for example there is no need to have the branch in the yaml file. @yueguoguo could you please confirm this?